config/locales/en.yml

```diff
@@ -108,6 +108,18 @@ en:
       unsubscribe: Unsubscribe
       username: Username
       web: Web
+    custom_emojis:
+      created_msg: Emoji successfully created!
+      delete: Delete
+      destroyed_msg: Emojo successfully destroyed!
```

Comment:
Typo 😭

Reply:
I did it on purpose as a nod to a popular joke on Mastodon. It's a rare occuring string, should be fine

Reply:
Ah, convinced 👍